framework/include/vectorpostprocessors/LeastSquaresFit.h

```diff
@@ -69,6 +69,14 @@ class LeastSquaresFit :
   /// The number of samples to be taken
   unsigned int _num_samples;
 
+  /// Values used to scale and or shift x and y data
+  ///@{
```

Comment:
I'm not entirely sure it works that way. I think it should be 
```
///@{ Values used to scale and or shift x and y data
...
///@}
```

i.e. comment inside the bracket